nvflare/apis/utils/job_utils.py

```diff
@@ -81,3 +81,9 @@ def convert_legacy_zipped_app_to_job(zip_data: bytes) -> bytes:
                     out_zip.writestr(info, content)
 
         return writer.getvalue()
+
+
+def load_job_def(from_path: str, def_name: str) -> bytes:
```

Comment:
Suggest 

`def load_job_bytes(job_root_path: str, job_name: str) -> bytes:`

Because I would expect load_job_def to return `Job` object.